.github/workflows/tests.yaml

```diff
@@ -475,11 +475,18 @@ jobs:
                   mkdir objdir-clone || true
 
             - name: Build Python REPL and example apps
+              # NOTE: the data-mode-check + check-failure-die is not 100% perfect as different
+              #
```

Comment:
nit, is there a reason for this extra space?

Reply:
Inadvertent enter press. Removed.